content/en/docs/collector/custom-auth.md

```diff
@@ -1,117 +0,0 @@
----
-title: "Building a custom authenticator"
```

Comment:
Generally speaking, when pages disappear, we should add an appropriate redirect rule. (Related: #635)

So, in this case, if a reader was interested in the content of this page (and the _Authentication_ section on the parent page), where should they go to next for pertinent info?

/cc @bogdandrutu @tigrannajaryan

Reply:
> Generally speaking, when pages disappear, we should add an appropriate redirect rule. (Related: #635)

I've opened #638 to track this.

Reply:
Thanks. I think https://github.com/open-telemetry/opentelemetry.io/issues/668 is the right link.

Reply:
Right, thanks!